packages/core/contracts/cross-chain-oracle/chain-adapters/README.md

```diff
@@ -0,0 +1,14 @@
+## Configuration Details
+
+Setting up contracts for each type of network are different.
+
```

Comment:
nice. shall we add the same block for other chains?

Reply:
Yeah we should but i wanted to kick it off here and align on structure of this doc